rust/Cargo.toml

```diff
@@ -20,7 +20,7 @@ anyhow = "1.0.52"
 solana-sdk = "1.10.32"
 borsh = "0.9.3"
 rand = "0.7.3"
-phoenix-types = { git = "https://github.com/Ellipsis-Labs/phoenix-types", branch = "master" }
+phoenix-types = { path = "../../phoenix-types" }
```

Comment:
note to fix this before merging

Reply:
Ah thank you. Reverted in the latest commit

Reply:
no worries. i stick this in basically all my PRs and leave comments like this (https://github.com/Ellipsis-Labs/phoenix/pull/249#discussion_r1069911937) just to make sure it gets reverted before merging. ideally we would have CI set up to yell at you before merging so we don't need these manual comments